src/collective/cover/setuphandlers.py

```diff
@@ -1,8 +1,21 @@
 # -*- coding: utf-8 -*-
+from collective.cover.config import PROJECTNAME
+from plone.dexterity.interfaces import IDexterityFTI
 from Products.CMFPlone import interfaces as Plone
 from Products.CMFQuickInstallerTool import interfaces as QuickInstaller
+from zope.component import queryUtility
 from zope.interface import implements
 
+import logging
+import pkg_resources
+
+try:
+    pkg_resources.get_distribution('plone.app.stagingbehavior')
+except pkg_resources.DistributionNotFound:
+    IStagingSupport = None
+else:
+    from plone.app.stagingbehavior.interfaces import IStagingSupport
```

Comment:
for me `else` in a `try` block just makes sense if I really want the exception to be raised, and treat it in another layer of the system.

but in this case I still don't use the `else` clause.. I try to arrange the lines I expect to raise the exception out of the `try` clause.